python/pyspark/ml/functions.py

```diff
@@ -486,6 +486,7 @@ def predict(inputs: np.ndarray) -> np.ndarray:
     data = np.arange(0, 1000, dtype=np.float64).reshape(-1, 4)
```

Comment:
`predict_batch_udf` should also be added to https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.ml.rst#functions